configure

```diff
@@ -95,7 +95,7 @@ find_program()
 try_cxx_argument()
 {
     info Checking whatever compiler accpets "$2"...
-    if $compiler -c $CXXOPTS $2 ${POST_CXXOPTS:-} testfile.cc -o testfile.o > /dev/null 2>&1; then
+    if $compiler -c $CXXOPTS $2 ${POST_CXXOPTS:-} $LDFLAGS ${POST_LDFLAGS:-} testfile.cc -o testfile.o > /dev/null 2>&1; then
```

Comment:
It's not that you need to add $LDFLAGS to the compilation test, it's that the compilation test should link as well as compile. So you need to remove `-c` and change `-o -testfile.o` to just `-o testfile`. See mconfig.Linux.sh file for example.

Reply:
(Though, it does make sense to include `${POST_LDFLAGS:-}` since we want to check that any automatically chosen option is compatible with user-supplied link flags - so that's a good idea!)

Reply:
Hold on a second, `${POST_LDFLAGS:-}` or `${LDFLAGS_POST:-}`? seems like I need to fix this thing.

Reply:
They should be named consistently. If it's CXXOPTS_POST then it should be LDFLAGS_POST!